src/App.js

```diff
@@ -1,29 +1,11 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
```

Comment:
- [ ] Kindly implement unit tests, as per [this](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=NOTE%3A%20Make%20sure%20to%20add%20test%20for%20all%20your%20components%20and%20functions.) requirement